.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.0.0
+2.6.4
```

Comment:
ruby 2.0 is outdated and unmaintained so I updated to a stable and maintained version. This is only for dev env but let me know if you'd rather this change to be removed.

Reply:
This is fine as we are running CI against ruby 2.6, so it would be good to bump dev to it as we should be supporting the latest.